src/main/java/net/bhardy/bizzo/billing/BillingPolicy.java

```diff
@@ -19,6 +19,7 @@
 
 import java.time.LocalDate;
 import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)